app/src/main/java/org/projectbuendia/client/ui/dialogs/ValidationException.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 The Project Buendia Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License.  You may obtain a copy
+// of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distrib-
+// uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+// specific language governing permissions and limitations under the License.
+
+package org.projectbuendia.client.ui.dialogs;
```

Comment:
I would suggest this class be in its own package 'exception', outer of 'ui' package.